### PR TITLE
new fetcher to contibution section

### PR DESCRIPTION
**Pull request by ghost**
new files

### CODE REVIEW

**Comment by ghost** [comment]
https://nolik2000.github.io/gci16.fossasia.org/

**Comment by hoangvanthien** [comment]
Hello @nolik2000, thanks for your PR, your idea is appreciated. But ensure that your new feature wouldn't affect other parts of our website. 

Furthermore, please generate contributors via GitHub API, not a static array. Since there may be more contributors in the future.

screen shot 2016-12-14 at 08 33 13

screen shot 2016-12-14 at 08 33 20

File: index.html [hunk 1]
```diff
@@ -498,6 +515,7 @@ <h2 class="white-header">FOSSASIA on social media</h2>
 
 <!-- Scripts -->
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```
Comment (hoangvanthien) [inline index.html]: Remove this line, it's redundant, jQuery has been imported.

File: stylesheets/contributors.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  -webkit-animation-iteration-count: infinite;
+  animation-name: change;
+  animation-duration: 6s;
+  animation-iteration-count: infinite;
+}
+
+ul li {
+  line-height: 40px;
+  margin: 0;
+}
```
Comment (hoangvanthien) [inline stylesheets/contributors.css:68]: You need to indicate explicitly that these CSS rules only will be applied to `contributorsThanks` section.

File: javascripts/contributors.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      "Codethejason",
+      "Shubham-padia",
+      "Hesapadim",
+      "Animeshsinghweb",
+      "Dannymout",
+      "Royalharsh",
+      "Hesham-Burawi"
+    ],
+    i = 0;
+
```
Comment (hoangvanthien) [inline javascripts/contributors.js:35]: Do not use static array, use GitHub API provided for developers instead.

File: index.html (hunk 1 above)
Comment (hoangvanthien) [inline index.html]: Delete this line, it's redundant, jQuery has been imported.

**Review by hoangvanthien — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Thanks, I will  fix it

**Comment by abishekvashok** [comment]
K happy coding :)

**Review by ghost — COMMENTED** [review]
featcher

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix as requested.

**Pull request closed without merging**